db/migrate/20240319020304_create_tradings.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTradings < ActiveRecord::Migration[7.1]
+  def change
+    create_table :tradings do |t|
+      t.float :amount
```

Comment:
maybe we can set a default value for amount?

Reply:
Bhaiya, when someone will add income or expense, he must explicitly define the amount related to it. If we set default as zero, will it be a good record where recorder doesn't know how much transaction is involved in it? asking for your opinion bhaiya